pages/blog/release-notes-2.4.0.md

```diff
@@ -0,0 +1,113 @@
+---
+title: AsyncAPI Spec 2.4.0 Release Notes
+date: 2022-04-25T19:00:00+01:00
+type: Communication
+tags:
+  - Specification
+  - Release Notes
+cover: /img/posts/release-notes-2.4.0/cover.webp
+authors:
+  - name: Sergio Moya
+    photo: /img/avatars/smoya.webp
+    link: https://twitter.com/smoyac
+    byline: Pilot at AsyncAPI Airlines
+excerpt: 'AsyncAPI 2.4 is now released. This brings ... TBD ...'
+featured: true
+---
+
+The new version of the AsyncAPI specification - 2.4.0 - is now available.
+
+> This is a minor release, and it doesn't bring any breaking changes. You can switch to it by modifying the following value in your AsyncAPI file `asyncapi: '2.3.0'` into `asyncapi: '2.4.0'`
+
+## Server Variables can be now referenced from components
+
+To allow for more flexibility in how AsyncAPI documents are structured and enable content to be reused, `serverVariables` can now be defined as reusable components.
+
+For example:
+
+```yaml
+asyncapi: 2.4.0
+servers:
+  development:
+    $ref: '#/components/servers/myserver'
+  production:
+    $ref: '#/components/servers/myserver'
+components:
+  servers:
+    myserver:
+      url: "{stage}.my-server.com:{port}"
+      protocol: ws
+      variables:
+        stage:
+          $ref: "#/components/serverVariables/stage"
+        port:
+          $ref: "#/components/serverVariables/port"
+  serverVariables:
+    stage:
+      default: dev
+    port:
+      enum: [5000, 6000]
+      default: 5000
+```
+
+These are added to the many other aspects of the AsyncAPI specification which can be declared as reusable components. You can see the full list in the [Components Object section of the AsyncAPI specification](https://www.asyncapi.com/docs/specifications/v2.4.0#componentsObject).
+
+This new feature was contributed by [Daniel Kocot](https://github.com/danielkocot). For more detail, see this [pull request](https://github.com/asyncapi/spec/pull/717) and the [Github issue where this change was discussed](https://github.com/asyncapi/spec/issues/707).
+
+## Security can now be defined at Operation level
```

Comment:
@sekharbans-ebay would you like to review the release notes I wrote here about your recent contribution? Also check the example in detail, please.
Thanks!